src/main/java/org/jabref/logic/importer/fileformat/WorldcatImporter.java

```diff
@@ -0,0 +1,175 @@
+package org.jabref.logic.importer.fileformat;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.jabref.logic.importer.Importer;
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.logic.importer.fetcher.WorldcatFetcher;
+import org.jabref.logic.net.URLDownload;
+import org.jabref.logic.util.FileType;
+import org.jabref.logic.util.StandardFileType;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+public class WorldcatImporter extends Importer {
+
+	private String WORLDCAT_READ_URL;
+
+	private final static String NAME = "WorldcatImporter";
+	private final static String DESCRIPTION = "Takes valid XML from Worldcat Open Search and parses them to BibEntry";
+
+	public WorldcatImporter(){
+		//Used the same key as Worldcat fether
+		this.WORLDCAT_READ_URL = "http://www.worldcat.org/webservices/catalog/content/{OCLC-NUMBER}?recordSchema=info%3Asrw%2Fschema%2F1%2Fdc&wskey=" + WorldcatFetcher.API_KEY;
```

Comment:
I saw it should be able to get JSON from the worldcat api:
Could you please test that?
That would improve parsing a lot (instead of using the old XML)
https://help.oclc.org/Discovery_and_Reference/WorldCat_Discovery/Troubleshooting/Using_the_WorldCat_Search_API_documentation_indicates_a_JSON_response_But_even_though_I_am_sending_an_Accept_header_with_the_value_application_JSON_I_keep_getting_the_XML_response